configs/platforms/debian-11-aarch64.rb

```diff
@@ -0,0 +1,15 @@
+platform "debian-11-aarch64" do |plat|
+  # Delete the lines below when a vanagon with Debian 11 support is released
+  plat.servicedir "/lib/systemd/system"
```

Comment:
It seems Vanagon with Debian 11 is released so we can remove all default.
check this out [debian-11-aarch64](https://github.com/puppetlabs/vanagon/blob/main/lib/vanagon/platform/defaults/debian-11-aarch64.rb)

Reply:
Thanks Maqsood, I have updated it accordingly